extensions/tekton/examples/src/main/java/io/fabric8/tekton/api/examples/CancelTask.java

```diff
@@ -0,0 +1,31 @@
+package io.fabric8.tekton.api.examples;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Component;
+import io.fabric8.knative.internal.pkg.apis.Condition;
+import io.fabric8.tekton.client.DefaultTektonClient;
+import io.fabric8.tekton.client.TektonClient;
+import io.fabric8.tekton.pipeline.v1beta1.TaskRun;
+
+public class CancelTaskRun {
```

Comment:
I'm not sure if you tested this or not, but this is not compiling:
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project tekton-examples: Compilation failure: Compilation failure: 
Error:  /home/runner/work/kubernetes-client/kubernetes-client/extensions/tekton/examples/src/main/java/io/fabric8/tekton/api/examples/CancelTask.java:[11,8] class CancelTaskRun is public, should be declared in a file named CancelTaskRun.java
Error:  -> [Help 1]
```

Reply:
Thanks @rohanKanojia I had to make some adjustments from the version I had tested to put it into the format for consumption as an example.